src/bitcoind.cpp

```diff
@@ -23,6 +23,12 @@
 
 #include <stdio.h>
 
+// default bitcoin.conf file template
+const char* g_default_conf_file =
+#include <bitcoin.conf>
+    ;
```

Comment:
I don't think this is a good way to do this. I'd prefer rename to `bitcoin_conf.h` and move `const char ...` into that file.

Reply:
The R() is for being able to `#include` the file. But I like your idea better. I will update. Thanks for the suggestions.

Reply:
Also sorry for not explaining further. R() is for multiline string literal: https://stackoverflow.com/a/20508617/2302781